README.md

```diff
@@ -291,7 +291,7 @@ downloadBase64({
 expandedTitle</kbd><br/> <kbd>App version >= 14.8: Additional properties and
 deprecations</kbd><br/> <kbd>Partial support in B2P App version <=24.10:
 title</kbd><br/> <kbd>Partial support in B2P App version >=24.11: right
-actions</kbd><br/> <kbd>Full support in B2P App version >=24.12: title</kbd>
```

Comment:
This is just a quick fix as in B2P we are not going to implement the full version.